frontend/src/pages/RestaurantDetailsPage.tsx

```diff
@@ -33,6 +33,7 @@ export default function RestaurantDetailsPage() {
     return (
         <DefaultPageTemplate pageTitle={restaurant.title}>
             <p>{restaurant.city}</p>
+            <Button href={`/restaurants/edit/${id}`}>Edit</Button>
             <Button href="/">Back</Button>
```

Comment:
It's probably better to navigate via react-router-dom instead of using href

Reply:
This href prop is calling react router in the Button component.